drivers/volume/portworx/portworx.go

```diff
@@ -55,7 +55,7 @@ const (
 	validateReplicationUpdateTimeout = 10 * time.Minute
 	validateClusterStartTimeout      = 2 * time.Minute
 	validateNodeStartTimeout         = 3 * time.Minute
-	validatePXStartTimeout           = 5 * time.Minute
+	validatePXStartTimeout           = 20 * time.Minute
```

Comment:
Do we have logic in torpedo to pass this down validatePXStartTimeout  from Jenkins job as parameter to test ? Lets keep the default value as 5 mins so that default tests does not change. Only the tests like sybench would change this to 20 mins for now.

Reply:
Agreed. We cannot bump the default timeout to 20. This needs to be parameterized.

Reply:
Thiago where are the changes for the parameterizing the timeout ???